src/main/java/time.java

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
An interesting way to format the Date and Time. Could consider using the `DateFormat` class to make the formatting simpler with `DateFormat dateFormat = new SimpleDateFormat("dd/MM/yyyy HHmm")`